easybuild/easyblocks/m/mcr.py

```diff
@@ -147,3 +145,13 @@ def make_module_extra(self):
         txt += self.module_generator.set_environment('MCRROOT', os.path.join(self.installdir, self.subdir))
 
         return txt
+
+    def find_subdir(self):
+        # determine subdirectory (e.g. v84 (2014a, 2014b), v85 (2015a), ...)
+        import glob
```

Comment:
please move this `import` statement to the top of the easyblock (and keep `import` lines sorted alphabetically)